build.sh

```diff
@@ -218,6 +213,34 @@ fi
 # Replace spaces with semicolons in SKBUILD_EXTRA_CMAKE_ARGS
 SKBUILD_EXTRA_CMAKE_ARGS=$(echo ${EXTRA_CMAKE_ARGS} | sed 's/ /;/g')
 
+# Build and install libucxx_python.so
+if buildAll || hasArg libucxx_python; then
+    CMAKE_GENERATOR="${CMAKE_GENERATOR:-Ninja}"
+    pwd
+    cmake -S $REPODIR/cpp/python -B ${PYTHON_BUILD_DIR} \
+          -G${CMAKE_GENERATOR} \
+          -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX} \
+          -DDISABLE_DEPRECATION_WARNINGS=${BUILD_DISABLE_DEPRECATION_WARNINGS} \
+          -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
+          -DCMAKE_EXPORT_COMPILE_COMMANDS=${BUILD_COMPILE_COMMANDS} \
+          ${EXTRA_CMAKE_ARGS}
+
+    cd ${PYTHON_BUILD_DIR}
+
+    compile_start=$(date +%s)
+    cmake --build . -j${PARALLEL_LEVEL} ${VERBOSE_FLAG}
+    compile_end=$(date +%s)
+    compile_total=$(( compile_end - compile_start ))
```

Comment:
Do we typically report times like this?

Reply:
This and the `pwd` were both copied and pasted from another part of the code. Nothing is even printing `compile_total`, so I've removed this and the `pwd` from both places.